packages/api/jest.config.js

```diff
@@ -6,4 +6,12 @@ module.exports = {
   moduleNameMapper: {
     "\\.(css)$": "identity-obj-proxy",
   },
+  // Clears all mock usage data and removes the mocked implementation, resetting it to its original no-op state.
+  resetMocks: true,
```

Comment:
`resetMocks` can be removed, because `restoreMocks` does everything `resetMocks` does

Reply:
Removed